cmake/android/tester/app/build.gradle.in

```diff
@@ -11,12 +11,19 @@ android {
         versionName "1.0"
         testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
     }
+
     buildTypes {
         release {
             minifyEnabled false
             proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
         }
     }
+
+    externalNativeBuild {
+        cmake {
+            version "3.10.2"
```

Comment:
```suggestion
            version "3.9.0+"
```
But, as I see, the tests runner are not using Min CMake version at all... So, we can rely on default CMake 3.6.